gammapy/spectrum/extract.py

```diff
@@ -134,6 +134,7 @@ def process(self, observation, bkg):
             backscale=1,
             backscale_off=bkg.a_off,
             obs_id=observation.obs_id,
+            gti=observation.gti,
```

Comment:
This case requires a test. You could make sure that the correct GTI table is present in the existing tests.